### PR TITLE
Update animations.md

### PR DESCRIPTION
**Pull request by ehsan-karamad**
Revised the animations policy to propose a modified policy that blocks layout inducing animations as opposed to the non-composited animations.

The changed is motivated discussions in issues #202, #203, and #204.

### CODE REVIEW

File: policies/animations.md [hunk 1]
```diff
@@ -1,86 +1,92 @@
...
-
-One way to avoid this scenario is to move the animations off of the CPU
-entirely. Modern desktop computers and mobile devices have powerful GPUs that
-can perform certain kinds of animations much more efficiently, and can leave the
-CPU free to handle other things, like responding to user input.
-
-Deeper explanation
+In order to produce animations on the web, developers declare transitions of style
+in either CSS (e.g., `@keyframes` and `animation`) or JavaScript (e.g., `element.animate()`). 
+In principal, these transitions can be used to animate just about any CSS property (
```
Comment (clelland) [inline policies/animations.md]: Minor nit -- can you move the "(" to the next line to avoid unnecessary whitespace after it?

File: policies/animations.md [hunk 2]
```diff
@@ -1,86 +1,92 @@
...
+backgrounds, sizes, margins, fonts, borders, positions amongst many other).
+
+In order to support such animations, the browser has to constantly update the values for
+all of these properties, and then calculate all their effects on the page; ideally, this has
+to has to be done sixty times per second.
+
+When properties such as `size` or `position` are changed in an animation, many other 
+elements on the page to have to be moved around constantly (*layout and re-layout*), and 
+browsers can struggle to keep up. When that happens, the result is a potentially stuttering
+animation and a poorly performing and non-responsive web page; sometimes even the entire page
```
Comment (clelland) [inline policies/animations.md]: >the entire page slows down

I realize that I missed a word in the original text -- I think I intended to say that in the worst case, the entire *machine* can slow down, as all CPU resources are devoted to the animation.

Comment (ehsan-karamad) [reply policies/animations.md]: page changed to *machine*

File: policies/animations.md (hunk 1 above)
Comment (ehsan-karamad) [reply policies/animations.md]: Done.
